### ISSUE TITLE
cannot download the excutables

### ISSUE THREAD

**Issue by KelvinChern** [HamishGBrown/MuSTEM#3]
Dear Hamish
It does not seem that we can download these pre-excutable anymore? People would really appreciate it if they could be updated. PS: I am trying to use it for simulations as well. Thank you so much.
Best wishes
Kelvin

**Comment by HamishGBrown** [comment]
Hi Kelvin,
Thanks for alerting me to this I've just updated the links in the readme.md file, so please try again. You can always access the executables by simply navigating to the [Executables](https://github.com/HamishGBrown/MuSTEM/tree/master/Executables) folder. Please let me know if this hasn't fixed your problem.

Cheers,
Hamish

**Comment by KelvinChern** [comment]
Hi Hamish 
Almost in a second, wow. Thank you very much.  I  will get you know soon
Best Wishes,
Kelvin

**Comment by KelvinChern** [comment]
yes, it is all there. I am sure many people are interested in it. Many thanks
